client-spark/spark3/src/main/java/org/apache/spark/shuffle/DelegationRssShuffleManager.java

```diff
@@ -123,6 +123,7 @@ private boolean tryAccessCluster() {
     Map<String, String> extraProperties = Maps.newHashMap();
     extraProperties.put(
         ACCESS_INFO_REQUIRED_SHUFFLE_NODES_NUM, String.valueOf(assignmentShuffleNodesNum));
+    extraProperties.putAll(RssSparkConfig.sparkConfToMap(sparkConf));
```

Comment:
We shouldn't put Spark configurations to coordinator. Coordinator isn't related to compute framework. The spark configurations is coupled with Spark framework. You can have an extractor instead of passing them directly.

Reply:
This PR is related to https://github.com/apache/incubator-uniffle/pull/2255 , I want to determine whether to ban an app by a specific `ban id` which extract from one of the spark client config, but I don't want client config it concretely.

Reply:
I'm not sure whether it is good design to have a ban id list. It seems that it isn't common enough.

Reply:
It's not common if bannedIdProvider contains spark.